### PR TITLE
Feature/errorhandling

### PR DESCRIPTION
**Pull request by willhobson85**
Created components to display when there is a 4XX or 5XX error, as well as redirect to "/" if user tries to go to an invalid Therapist page. Also created component to display when an invalid url is reached. 

# Pull Request
- [X] New feature (adds functionality)
- [ ] Refactor
- [X] Styling
- [ ] Bug Fixes

# Please describe the tests that you ran to verify your changes.
- [X] I tested my changes in the browser
- [ ] I tested with Cypress

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any other comments, questions or concerns?

Mob coded with @alinix1 @danalchapman @emilyjmiles

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -3,31 +3,36 @@ import LandingPage from '../LandingPage/LandingPage';
...
-
+  
   return (
     <main className="app-main">
       <header className="app-header">
         <h1 >Equilibrium</h1>
       </header>
-      <Switch >
+      {error
+      ? <ErrorMessage error={error} />
```
Comment (AshRing) [inline src/Components/App/App.js:22]: It looks like this is intended to be a full-page error - I suggest making an error route and directing the user there if there is an error. This way it is scalable if any other API requests are added, you can send them to the error page.

You can utilize `useEffect` to listen for the `error` value to change, and redirect the user when it does. For example --
```
const navigate = useNavigate();
useEffect(() => {
  if(error) {
    navigate("/error", { state: { error }});
  }
});
```

**Review by AshRing — CHANGES_REQUESTED** [review]
I just have the one suggestion around the error placement, but otherwise this looks great, good work.

**Comment by willhobson85** [comment]
Update route with @AshRing to get the error page listed as a full page, instead of taking over the landing page.

**Review by AshRing — APPROVED** [review]
🚀

**Pull request merged**